src/components/menu/item.tsx

```diff
@@ -55,6 +60,11 @@ const MenuItem: FC<Props> = ({
           <IconComponent width="24px" height="24px" />
         </span>
         {title}
+        {NotificationComponent ? (
+          <span className="w-12/12 text-right text-salmon px-5">
```

Comment:
Not sure about the use of the `text-salmon` style here. Can you elaborate on the use of that style?

Reply:
Thanks for noticing this Virginia ! At the beginning my notification was black in DH without this class here, but it is no more the case so I removed it